From 184cbdd5534bdae644d62e7ae9e36989d17c0b53 Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Sun, 15 Jun 2025 12:31:38 -0400
Subject: [PATCH] Support environment in MMC when booted in FEL mode

---
 board/sunxi/board.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/board/sunxi/board.c b/board/sunxi/board.c
index c7a2205ed61..d6b9243dc08 100644
--- a/board/sunxi/board.c
+++ b/board/sunxi/board.c
@@ -179,6 +179,8 @@ enum env_location env_get_location(enum env_operation op, int prio)
 	if (prio == 0) {
 		if (IS_ENABLED(CONFIG_ENV_IS_IN_FAT))
 			return ENVL_FAT;
+		if (IS_ENABLED(CONFIG_ENV_IS_IN_MMC))
+			return ENVL_MMC;
 		if (IS_ENABLED(CONFIG_ENV_IS_IN_UBI))
 			return ENVL_UBI;
 	}
-- 
2.43.0

